Source/svg/SVGParser.swift

```diff
@@ -1189,10 +1189,35 @@ open class SVGParser {
     }
 
     fileprivate func getDoubleValue(_ element: SWXMLHash.XMLElement, attribute: String) -> Double? {
-        guard let attributeValue = element.allAttributes[attribute]?.text, let doubleValue = Double(attributeValue) else {
+        guard let attributeValue = element.allAttributes[attribute]?.text else {
             return .none
         }
-        return doubleValue
+        return doubleFromString(attributeValue)
+    }
+    
+    fileprivate func doubleFromString(_ string: String) -> Double? {
+        if let doubleValue = Double(string) {
+            return doubleValue
+        }
+        do {
+            let matcher = try NSRegularExpression(pattern: "([a-zA-Z]+)", options: .caseInsensitive)
```

Comment:
`NSRegularExpression` should be created once for the parser, not for every value.